js-miniapp-bridge/src/bridge.js

```diff
@@ -96,6 +97,21 @@ MiniAppBridge.getUniqueId = function() {
     );
   });
 };
+
+/**
+ * Associating requestPermission function to MiniAppBridge object
+ * @param {String} requestType Type of permission that is requested. For eg., location
+ */
+MiniAppBridge.requestPermission = function(requestType) {
```

Comment:
Maybe `requestType` should be named something like `permission` instead? `requestType` is a bit confusing because it puts emphasis on the `request` part rather than the `permissions`.